app-backend/api/src/main/scala/scene/Routes.scala

```diff
@@ -44,6 +46,19 @@ trait SceneRoutes
 
   implicit val ec: ExecutionContext
 
+  val rasterIOContext = ExecutionContext.fromExecutor(
+    Executors.newFixedThreadPool(
+      4,
+      new ThreadFactoryBuilder().setNameFormat("geotiffinfo-%d").build()
+    )
+  )
+
+  implicit val rasterIOContextShift = IO.contextShift(rasterIOContext)
```

Comment:
Should this be implicit? I think this could lead to accidentally using this in places besides retrieving geotiff info

Reply:
An implicit context shift was still required for `IO.start`, so I asked for one that happens to share the same name as the one defined in `Main.scala`. Logging is still coming from the correct place in tests, so I believe this is now good to go.